### PR TITLE
Edit meeting

### PR DESCRIPTION
**Pull request by jc121F1**
Can now remove members.
Added testing.
Corrected HTML.

### CODE REVIEW

File: BookClub/models/__init__.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import imp
```
Comment (pleofthegem) [inline BookClub/models/__init__.py:1]: why was this added?

Comment (jc121F1) [reply BookClub/models/__init__.py:1]: Mistake. Removing now.

**Review by pleofthegem — APPROVED** [review]
looks good

**Review by amir-rahim — APPROVED** [review]
